api/v1alpha1/envoygateway_types.go

```diff
@@ -151,6 +151,10 @@ type ExtensionAPISettings struct {
 	// EnableEnvoyPatchPolicy enables Envoy Gateway to
 	// reconcile and implement the EnvoyPatchPolicy resources.
 	EnableEnvoyPatchPolicy bool `json:"enableEnvoyPatchPolicy"`
+
+	// EnableEnvoyExtensionPolicy enables Envoy Gateway to
```

Comment:
@envoyproxy/gateway-maintainers @envoyproxy/gateway-reviewers thoughts on this ?

Reply:
I'm fine to remove all these, it will not happen unless you create a CR.